boards/arm/nucleo_f207zg/nucleo_f207zg.dts

```diff
@@ -19,15 +19,15 @@
 
 	leds {
 		compatible = "gpio-leds";
-		green_led_1: led@0 {
+		green_led_1: led_1 {
 			gpios = <&gpiob 0 GPIO_INT_ACTIVE_HIGH>;
```

Comment:
According to the DT specification:
```
If the node has no reg property, the @unit-address must be omitted and the node-name alone
differentiates the node from other nodes at the same level in the tree.
```
The node-name here is `led_1` for all nodes.

Reply:
@ydamigos is this resolved?